SPEC.md

```diff
@@ -59,7 +61,40 @@ There is currently only one supported request type, `put`, which uploads a CAR f
 A `put` request description must contain a `rootCID` field whose value is the root CID of a Content Archive included in the request body.
 The CID should be encoded as a CIDv1 string.
 
-The `put` object also contains a `tags` key/value map that may contain arbitrary metadata tags. Currently this is used to identify the target blockchain (always `solana`) and the cluster that the user intends to mint on (e.g. `devnet` or `mainnet-beta`). Unrecognized tags will be discarded by the backend, and tags should not be used to store arbitrary metadata. Future revisions to this spec may introduce additional tags.
+The `put` object also contains a `tags` key/value map that may contain arbitrary metadata tags. Currently accepted tags are listed below:
+
+##### `chain`
+
+Indicates the blockchain that will be used for minting. Currently the only valid value is `"solana"`.
+
+##### `solanaCluster`
+
+Indicates which [Solana cluster](https://docs.solana.com/clusters) will be used for minting. Must be provided when `chain == "solana"`. Acceptable values are: `"mainnet-beta"`, `"devnet"`, `"testnet"`.
+
+**Note:** an earlier draft of this spec & library used the key `solana-cluster` for this tag. This was changed to "camel case" for consistency and to play nice with JavaScript conventions. The original key will be supported on the backend for a short time, but support will be removed in early 2022.
+
+##### `mintingAgent`
+
+The `tags` map MUST include a `mintingAgent` tag, whose value should identify the tool or platform used to prepare the upload.
```

Comment:
@dchoi27 does this make sense re: the recommended values?

Reply:
yup sgtm!